core/src/main/java/hudson/model/DirectoryBrowserSupport.java

```diff
@@ -446,6 +452,17 @@ public long getSize() {
             return size;
         }
 
+        public long getLastModified() {
```

Comment:
Missing `@since TODO` on all additions of public APIs.

Reply:
Also needs documentation of the output format. Like this one: https://docs.oracle.com/javase/7/docs/api/java/io/File.html#lastModified()

Reply:
ok